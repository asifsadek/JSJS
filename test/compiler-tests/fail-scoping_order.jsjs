val x = 5;

print(x + y);

val y = 10;
