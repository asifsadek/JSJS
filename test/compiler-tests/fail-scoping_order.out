Error: value 'y' was used before it was defined
